test/fixtures/plugins.js

```diff
@@ -0,0 +1,19 @@
+exports.valid = {
+  dependencies: {
+    'rnpm-plugin-link': '1.0.0',
+  },
+  devDependencies: {
+    'rnpm-plugin-build': '1.0.0',
+  },
+};
+
+exports.duplicates = {
+  dependencies: {
+    'rnpm-plugin-link': '1.0.0',
+  },
+  devDependencies: {
+    'rnpm-plugin-link': '1.0.0',
+  },
+};
+
+exports.empty = {};
```

Comment:
These are some explicit `package.json`'s we may use in tests, or define our own inline.